### PR TITLE
Guides

### PR DESCRIPTION
**Pull request by NikBombadil**
Ripped info from the main wiki and organized the info. Next step is to add flavour to the text and correct any grammatical errors.

### CODE REVIEW

File: docs/MatterOverdrive/Guides/Items/Tools/empty [hunk 1]
```diff
@@ -0,0 +1 @@
+##empty
```
[the remark this answers is not in the document]
Comment (NikBombadil) [reply docs/MatterOverdrive/Guides/Items/Tools/empty]: Good to know! Honestly just didn't know how to PR single items, will look into it though.

**Comment by TheCodedOne** [comment]
All of this data is wrong...

File: docs/MatterOverdrive/Guides/Blocks/Machines/Matter Analyzer [hunk 2]
```diff
@@ -0,0 +1,5 @@
+The Matter Analyzer, is the block equivalent to the Matter Scanner. The main difference is, the Analyzer is the only way to scan items. It also scans items and blocks twice as much as the Matter Scanner, increasing their progress by 20%.
+How it works:
+
+The Matter Analyzer requires power, in order to function. When an item is placed in the input slot, the analyzer will search for valid destinations. It will need to find a valid  to use as storage. The drives will be housed in Pattern Storage. The Analyzer must be connected to a Matter Network to search for free pattern storage. It can only be connected trough it’s back side. The Analyzer is smart enough to hold progression when pattern storage is not found. A Network Flash Drive can be used to limit the pattern storage blocks the Analyzer can interact with, by putting it in the network filter slot in the configurations tab.
```
Comment (Dev-Osmium) [inline docs/MatterOverdrive/Guides/Blocks/Machines/Matter Analyzer]: A lot of the flow here doesn't make sense

File: docs/MatterOverdrive/Guides/Blocks/Machines/Pattern Monitor [hunk 3]
```diff
@@ -0,0 +1,3 @@
+The Pattern Monitor is used to issue Item Pattern Task to Replicator. It searches the local network for Pattern Storage, and displays all item patterns. Then an amount is chosen to be sent to the replicators.
+
+To send the request, press the Second button on the right.
```
Comment (Dev-Osmium) [inline docs/MatterOverdrive/Guides/Blocks/Machines/Pattern Monitor]: This should probably be a description of what the button looks like, with a picture

**Pull request closed without merging**